app/misc/authenticate.ts

```diff
@@ -80,6 +80,12 @@ function ModalSignIn(callback) {
         getUserInfo(callback);
 }
 
+// provide a fresh cred object. 
+// previously we were trying to use same object again and again which was cauing issues
+function getCredentials(){
+  //Note: might need to update with Oauth
+  return Git.Cred.userpassPlaintextNew(getUsernameTemp(), getPasswordTemp());
```

Comment:
I would suggest syncing up with @dragonprevost so that we don't come across any hiccups once we merge.